src/components/Pivot/styles.scss

```diff
@@ -121,72 +121,6 @@ section.virtualized-pivot{
 	  transition: all 0.3s cubic-bezier(.25,.8,.25,1);
 	}
 
-	section.pivot-grid{
```

Comment:
@turnerniles It seemed like this was all duplicated css from `Table/styles.scss`